models/articles.js

```diff
@@ -18,3 +18,28 @@ exports.selectAllArticles = () => {
     return rows;
   });
 };
+
+
+
+exports.selectArticle = (article_id) => {
+
+  const sql = ` SELECT article_id FROM articles`;
+  return db
+    .query(sql)
+    .then(({ rows }) => {
+      const idWhiteList = rows.map((row) => row.article_id);
```

Comment:
For the error handling here, we wouldn't need to create a white list. You can just promise reject with a 404 error if the row count is empty. 👍